ansible_collections/arista/avd/roles/eos_validate_state/python_modules/tests/avdtestsecurity.py

```diff
@@ -0,0 +1,41 @@
+# Copyright (c) 2023 Arista Networks, Inc.
+# Use of this source code is governed by the Apache License 2.0
+# that can be found in the LICENSE file.
+from __future__ import annotations
+
+from functools import cached_property
+
+from ansible_collections.arista.avd.plugins.plugin_utils.eos_validate_state_utils.avdtestbase import AvdTestBase
+from ansible_collections.arista.avd.plugins.plugin_utils.utils import get
+
+
+class AvdTestHttpsSSL(AvdTestBase):
+    """
+    AvdTestHttpsSSL class for HTTPS SSL tests.
+    """
+
+    anta_module = "anta.tests.security"
+    categories = ["ssl"]
+    description = "eAPI HTTPS server SSL profile is configured and valid"
+
+    @cached_property
+    def test_definition(self) -> dict:
+        """
+        Generates the proper ANTA test definition for all SSL tests.
+
+        Returns:
+            test_definition (dict): ANTA test definition.
+        """
+        anta_tests = []
+        profile = get(self.hostvars[self.device_name], "custom_structured_configuration_management_api_http.https_ssl_profile")
```

Comment:
you should not be reading in `custom_structured_configuration` as this is not part of the rendered structured config
```suggestion
        profile = get(self.hostvars[self.device_name], "management_api_http.https_ssl_profile")
```

Reply:
Please use the helper functions of `AvdTestBase` to add logging. You could do something like:
```
if (profile := self.logged_get(key="management_api_http.https_ssl_profile", logging_level="INFO") is None:
    return None
``` 
Check `AvdTestMLAG` for reference.